### PR TITLE
if jsonfield is serialized as str reload as json

### PR DESCRIPTION
**Pull request by haricot**
related https://github.com/divio/djangocms-history/pull/22, #234

### CODE REVIEW

File: cmsplugin_cascade/models_base.py [hunk 1]
```diff
@@ -103,7 +104,14 @@ def sanitize_children(self):
             for child in children:
                 child.save(sanitize_only=True)
                 child.sanitize_children()
-
+                
+    @classmethod
+    def from_db(cls, db, field_names, values):
+        instance = cls(*values)
+        if isinstance(instance.glossary, str):
+           instance.glossary=json.loads(instance.glossary)
```
Comment (jrief) [inline cmsplugin_cascade/models_base.py]: not PEP8 conformant ` = ` requires spaces.

**Comment by jrief** [comment]
I'm sure this happened to you, but from my point of view, this should be impossible to happen.
Is there a way to reproduce it?

**Comment by haricot** [comment]
For reproduce, just install djangocms-history, and use undo redo, publish. it will block the database.

possible related:
https://code.djangoproject.com/ticket/27538
https://stackoverflow.com/questions/42771437/why-does-django-return-postgres-jsonfield-value-as-string
https://bitbucket.org/schinckel/django-jsonfield/issues/57/cannot-use-in-the-same-project-as-djangos

**Comment by haricot** [comment]
I think it's happening here:
https://github.com/divio/djangocms-history/blob/9f572e015697e9a6c6c44318ad1f39573d1b7fbe/djangocms_history/helpers.py#L73

or here for djangocms-transfer
https://github.com/divio/djangocms-transfer/blob/80c9e17b4197251b7c8fc184054fe9d23c27f2ec/djangocms_transfer/helpers.py#L47

**Pull request merged**